user/enterprise/custom-queues.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Enterprise Customer Worker Queues
+layout: en_enterprise
+---
+
+Setting up custom worker queues on Travis CI Enterprise allows your team to designate a particular worker (or group of workers) for building specific repositories. This is especially useful for differentiating workers that have been customized specially.
+
+Custom queues require [enabling two feature flags](#Enable-Queues-on-the-Platform). Once these flags are set, you can define the configuration for your custom queues in the [Management Console settings](#Define-Custom-Queues-in-the-Management-Console), and [allocate workers to the new queues](#Define-Custom-Queues-Settings-on-The-Workers)
```

Comment:
How about 

> Using custom queues....

or 

> To use custom queues, enable two feature....

Reply:
I went with:
> there are two features flags that need to be enabled for custom queues.

For the sake of not sounding exactly like the first paragraph, I though it was good to put "custom queues" at the end of the sentence.